### PR TITLE
Learn Python blog post

### PR DESCRIPTION
**Pull request by drj11**
Complete. Please review and comment as you see fit.

### CODE REVIEW

**Comment by Robadob** [comment]
It's a good read, seems thorough.

I think https://www.codefirstgirls.org.uk/ also run Python courses here (headed up by Heidi Christensen).

*I'm in the camp of learning Python as I need it, mostly for generating figures, usually a case of just searching for the correct syntax. Though I was very pleased to discover `.__dict__` the other day, I've wanted dynamic access to an object's member variables in other (static) languages previously.*

**Comment by drj11** [comment]
@robadob please merge when you see fit :)

**Comment by drj11** [comment]
@robadob please merge when you see fit :)

**Comment by Robadob** [comment]
I've no experience with Jenkins, thought I'd leave it for someone who
understands that's in order.

**Comment by drj11** [comment]
ok, @willfurnass ?

File: _posts/2019-08-22-learn-python.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+I learnt Python so long ago that my personal experience isn't
+that much use.
+
+So I used my Twitter network.
+You are all very helpful and
+[my tweet where I ask how you learnt
+Python](https://twitter.com/drjtwit/status/1163843466528473088)
+got a lot of useful replies and suggestions.
+
+[![a laptop showing some code, taken between the shoulders of
```
Comment (willfurnass) [inline _posts/2019-08-22-learn-python.md:29]: This pic is huuuge - scale to 50% width? (probably easiest with `` than markdown syntax)

File: _posts/2019-08-22-learn-python.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+people huddled coding](/assets/images/DGCali-72.jpg "Coding at
+Django Girls Cali:
+CC-BY-SA")](https://argentinaenpython.com/galeria/django-girls-cali/)
+
+I have to preface this a warning.
```
Comment (willfurnass) [inline _posts/2019-08-22-learn-python.md]: gr: 'this with a'

File: _posts/2019-08-22-learn-python.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+echoing Emily Gorcenski's recent tweet about starting college this year:
+
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr">Learning will be a lifelong process and the best thing you can do while there is learn how to learn. <a href="https://t.co/ban2Wd8l4y">https://t.co/ban2Wd8l4y</a></p>&mdash; Emily G is absolutely going to replace you (@EmilyGorcenski) <a href="https://twitter.com/EmilyGorcenski/status/1163343747125583872?ref_src=twsrc%5Etfw">19 August 2019</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Almost everyone mentions "something" plus:
+Google / Stack Overflow / the docs / YouTube / trial-and-error.
+No doubt that being able to research and discover
+solutions to a problem using
+the general web is a Force Multiplier.
```
Comment (willfurnass) [inline _posts/2019-08-22-learn-python.md:106]: gr: ','?

**Review by willfurnass — CHANGES_REQUESTED** [review]
Interesting that there were no mentions of tutorials/books that focus on/start with scientific Python (other than Software Carpentry, which is quite introductory).

Mention Code Clinic at the end?

Add

**Comment by willfurnass** [comment]
@drj11 Forgot to say that overall I very much like the post!

File: _posts/2019-08-22-learn-python.md (hunk 2 above)
Comment (drj11) [reply _posts/2019-08-22-learn-python.md]: doing

File: _posts/2019-08-22-learn-python.md (hunk 1 above)
Comment (drj11) [reply _posts/2019-08-22-learn-python.md:29]: will do.

File: _posts/2019-08-22-learn-python.md (hunk 3 above)
Comment (drj11) [reply _posts/2019-08-22-learn-python.md:106]: where do you suggest using this comma? Oh you don't like the that my second sentence doesn't have a verb? :)

**Comment by drj11** [comment]
I agree, interesting that no one mentioned the scientific Python stack at all.
One of the next steps may to be more deliberately explore this part of Python.

File: _posts/2019-08-22-learn-python.md (hunk 1 above)
Comment (drj11) [reply _posts/2019-08-22-learn-python.md:29]: cropped and scaled

**Comment by drj11** [comment]
Added a mention a mention of Code Clinic, please resolve as you see fit.

**Review by willfurnass — APPROVED** [review]
BTW I _love_ your use of semantic line endings here!

**Pull request merged**